src/plugins/discover/public/application/components/context_app/context_app_legacy.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../../../../core/public/mixins';
+
+.dscDocsPage {
+  @include kibanaFullBodyHeight(54px); // action bar height
+}
+
+.dscDocsContent {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.dscDocsGrid {
+  flex: 1 1 100%;
+  overflow: auto;
+
+  .euiDataGridRowCell.euiDataGridRowCell--firstColumn {
```

Comment:
@Dmitriynj would it be possible to use a custom (`.dsc____`) class here? We try to avoid target `.eui` classes directly whenever possible.

Reply:
We need to set `padding: 0` to a first column cells directly, so if you would like to use custom class we have set it through `setCellProps({ className: 'dsc___' })`. [Here](https://github.com/elastic/kibana/pull/90906/files#r573425628) is similar usage of `.eui` classes for discover page

Reply:
After seeing the reference you linked I think it's fine to leave this as you currently have it.